surveys/2022-12/questions.md

```diff
@@ -0,0 +1,135 @@
+# Survey December 2022
```

Comment:
We should probably get this survey out soon 😄

Reply:
Haha yeah, I may have to update that :) 

a good timeline would be to 
- have it finished and published end of feb/start of march 
-  let it go a "month" until April/mid April
- review the feedback before Vancouver
- discuss about it during the Vancouver event

Reply:
I think what we have in the doc makes sense. I think asking for help to do the survey from the Foundation makes senses sooner than later and we can agree on the schedule with them (assuming they can help). @joesepi I know you helped with some past surveys, who at the Foundation do you think we should reach out to for help on the survey?